src/pages/en/guides/images.md

```diff
@@ -97,24 +97,23 @@ import logo from '../images/logo.png';
 
 ### `public/`
 
-Your images stored in `public/` can be used by components (`.astro`, `.mdx` and other UI frameworks) and also Markdown files (`.md`, `.markdown`).
+Your images stored in `public/` can be used by components (`.astro`, `.mdx` and other UI frameworks) and also Markdown files.
 
-The `src` attribute is **relative to the public folder**. These images can also be used in Markdown files using the `![]()` notation.
 
-Files in the `/public` directory are always served or copied as-is, with no processing. We recommend that local images are kept in `src/` when possible so that Astro can transform, optimize and bundle them.
+However, files in the `/public` directory are always served or copied as-is, with no processing. We recommend that local images are kept in `src/` when possible so that Astro can transform, optimize and bundle them.
```

Comment:
Doing this means you _can’t_ use the images in Markdown right? Do we want to call that out? If you have a Markdown-powered blog, we specifically would _not_ recommend images in `src/`, I guess.

```suggestion
However, files in the `/public` directory are always served or copied as-is, with no processing. If you are using images outside of Markdown files, we recommend that local images are kept in `src/` when possible so that Astro can transform, optimize and bundle them.
```

Reply:
I was wondering if the "when possible" was strong enough on its own, since the corresponding section on in `src` says that you can NOT use Markdown with your images there.

Reply:
“We recommend” is pretty strong. Do we think a reader has a clear picture of what “when possible” means? I’d be worried they miss the `src` comment — unless it’s perhaps really highlighted in a caution? But I like the goal here of getting rid of those!